### PR TITLE
Branch docs user guide

### PR DESCRIPTION
**Pull request by ruilingk**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -482,19 +480,22 @@ to assign tasks to modules straight away.
...
-Figure 15: TR4CKER after executing `edit 3 m/CS3241`
+Figure 15: TR4CKER after executing `edit 1 m/CS3241`
 
 3. Switch to the Module tab, and the task should appear under CS3241.
 ![assign module result](images/moduletab_assignmodule_result.png)
-Figure 16: TR4CKER updated Module tab after executing `edit 3 m/CS3241`
+Figure 16: TR4CKER updated Module tab after executing `edit 1 m/CS3241`
 
-As seen from *Figure 12* and *Figure 13* above, module codes tagged onto tasks will appear as a pink coloured tag under the
+As seen from *Figure 14* and *Figure 15* above, module codes tagged onto tasks will appear as a pink coloured tag under the
```
Comment (ethan-l-m-e) [inline docs/UserGuide.md:493]: Wow ty, completely missed it

**Review by ethan-l-m-e — APPROVED** [review]
looks good

File: docs/UserGuide.md [hunk 2]
```diff
@@ -529,9 +530,6 @@ Format: `modules del/INDEX`
 * The index **must be a positive integer** 1, 2, 3, …
 * Warning: This action is irreversible!
 
-Examples:
-* `modules del/1` - Deletes the 1st module in the modules list.
-
 1. You enter a command into the command box to delete a module, let's say module number 9 that we created previously.
 ![delete module before](images/moduletab_deletemodule_before.png)
 Figure 19: TR4CKER before executing `modules del/9`
```
Comment (cwenling) [inline docs/UserGuide.md:535]: Seems to have multiple figures labelled as figure 19?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -617,8 +618,8 @@ Format: `countdown del/INDEX`
 Figure 19: TR4CKER before executing `countdown del/2`
 
 3. Press enter, and TR4CKER will delete the event.
-![delete_after](images/countdowntab_delete_before.png)
-Figure 19: TR4CKER before executing `countdown del/2`
+![delete_after](images/countdowntab_delete_after.png)
+Figure 19: TR4CKER after executing `countdown del/2`
```
Comment (cwenling) [inline docs/UserGuide.md:622]: Here too

File: docs/UserGuide.md (hunk 2 above)
Comment (ruilingk) [reply docs/UserGuide.md:535]: i think ltr then standardise the figures bah

**Pull request merged**